core/cat/rabbit_hole.py

```diff
@@ -401,13 +412,10 @@ def __split_text(self, stray, text, chunk_size, chunk_overlap):
             "before_rabbithole_splits_text", text, cat=stray
         )
 
-        text_splitter = RecursiveCharacterTextSplitter.from_tiktoken_encoder(
-            chunk_size=chunk_size,
-            chunk_overlap=chunk_overlap,
-            separators=["\\n\\n", "\n\n", ".\\n", ".\n", "\\n", "\n", " ", ""],
-            encoding_name = "cl100k_base",
-            keep_separator=True,
-            strip_whitespace = True
+        text_splitter = self.text_splitter.from_tiktoken_encoder(
```

Comment:
what if one implements a splitter that doesn't have this method?

Reply:
I checked the code, this method is by default in the TextSplitter and since we accept any subclass of it, it must have that method.

Reply:
https://api.python.langchain.com/en/latest/base/langchain_text_splitters.base.TextSplitter.html#langchain_text_splitters.base.TextSplitter

Reply:
not sure this is the right strategy